.rubocop.yml

```diff
@@ -5,6 +5,9 @@ require:
 AllCops:
   NewCops: enable
 
+Style/HashSyntax:
+  EnforcedShorthandSyntax: either
```

Comment:
My understanding is that this allows both `foo: 'bar'` and `:foo => 'bar'`, is that right?

If so, I'd personally would rather we enforced one style or the other, for consistency. (FWIW, my favorite is `foo: 'bar').

Reply:
This is actually for checking the use of the explicit hash values -- which will also be reported for method calls with named parameters and this is why I had to use `either` 🙃 (`consistent`, which I used in another PR, failed for me here when I had a single parameter).

From [here](https://rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashSyntax):
<img width="262" alt="Screenshot 2023-08-01 at 10 35 41" src="https://github.com/Automattic/dangermattic/assets/2152244/610949f7-8f89-470c-99bd-39257a861648">

Reply:
Gotcha. Evidently, my understanding was wrong.